CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.16.0] - 2024-10-02
```

Comment:
For the sdk, all changes must be described in the Unreleased section. It's when a release is defined that the version number will be defined.
```suggestion
```